GOVERNANCE.md

```diff
@@ -0,0 +1,36 @@
+# Project governance
```

Comment:
this is outdated. I will request you to update these static files using controller-bootstrap. It is very easy and a single command that will update the files.

https://github.com/aws-controllers-k8s/controller-bootstrap/blob/main/README.md#getting-started